src/WordPress/Hooks.php

```diff
@@ -174,7 +174,8 @@ public function purgeCacheByRelevantURLs($postIds)
 
             // Don't attempt to purge anything outside of the provided zone.
             foreach ($urls as $key => $url) {
-                if (!Utils::strEndsWith(parse_url($url, PHP_URL_HOST), $wpDomain)) {
+                $url_to_test = is_array($url) ? $url['url'] : $url;
```

Comment:
you'll want to do a keys check here for url in the array otherwise you'll potentially end up with null.

Reply:
Thanks, I've done that and expanded it out to an `if` statement for extra clarity